Alarmi/Alarmi/Sources/ViewControllers/TodayViewController.swift

```diff
@@ -10,13 +10,110 @@ import UIKit
 
 class TodayViewController: UIViewController {
 
+    // MARK: View
+    
+    private let stackView: UIStackView = {
+        $0.translatesAutoresizingMaskIntoConstraints = false
+        $0.axis = .vertical
+        $0.distribution = .fill
+        $0.alignment = .leading
+        $0.spacing = 8
+        return $0
+    }(UIStackView())
+
+    private let descriptionLabel: UILabel = {
+        $0.translatesAutoresizingMaskIntoConstraints = false
+        $0.text = "마지막으로 전화한지 5일이 지났어요."
+        $0.font = UIFont.preferredFont(forTextStyle: .title2)
+        $0.adjustsFontSizeToFitWidth = true
```

Comment:
p3;

자동완성 잘못 누르신 것 같...
```suggestion
        $0.adjustsFontForContentSizeCategory = true
        $0.numberOfLines = 0
```
numberOfLines도 추가해야 합니다!

Reply:
37c3cf39384d81ed73ef24dad451514490b2d67f 반영했습니다